contracts/lifebanktest/src/lifebanktest.cpp

```diff
@@ -0,0 +1,35 @@
+#include <lifebanktest.hpp>
+
+ACTION lifebanktest::hi(name from, string message, string message2) {
```

Comment:
@leisterfrancisco  what are you using this action for?

Reply:
This shouldn't be uploaded